src/ape_ethereum/ecosystem.py

```diff
@@ -152,12 +152,19 @@ def as_dict(self):
 class Receipt(ReceiptAPI):
     def raise_for_status(self, txn: TransactionAPI):
         """
-        Raises :class`~ape.exceptions.OutOfGasError` when the
-        transaction failed and consumed all the gas.
+        Raise an error for the given transaction if the transaction has failed.
 
-        Raises :class:`~ape.exceptions.TransactionError`
-        when the transaction has a failing status otherwise.
+        Args:
+            txn (:class:`ape.api.providers.TransactionAPI`): The sent-transaction to
+              check the status of.
```

Comment:
```suggestion
            txn (:class:`ape.api.providers.TransactionAPI`): The status of the sent-transaction to
              check.
```

Reply:
Was trying to not end in 'of', but I think I changed the meaning/intention of the comment so please disregard.

Reply:
Ah, okay, well maybe something like:

```
The transaction to check
```